.make.defaults

```diff
@@ -179,6 +181,16 @@ __check_defined = \
 	-find . -name '*.egg-info' | xargs rm -rf
 	-find . -name '__pycache__' | xargs rm -rf
 	-rm -rf dist
+	-rm -rf $(REPOROOT)/artifacts > /dev/null 2>&1
+
+.PHONY: .defaults.docker-save-image
+.defaults.docker-save-image:
```

Comment:
should we add a rule to the `make clean` to clean up the ARTIFACTS_DIR, the images can take up a lot of space.

Reply:
Yes the removal of ARTIFACTS_DIR is in make clean target.